requirements/test.txt

```diff
@@ -103,17 +110,17 @@ edx-celeryutils==1.2.1
     # via
     #   -r requirements/base.txt
     #   super-csv
-edx-django-utils==4.5.0
+edx-django-utils==5.0.0
```

Comment:
https://github.com/openedx/edx-django-utils/releases/tag/5.0.0
Couldn't find the mentioned keys being used in this repo.